website/content/commands/config/delete.mdx

```diff
@@ -7,10 +7,33 @@ page_title: 'Commands: Config Delete'
 
 Command: `consul config delete`
 
+Corresponding HTTP API Endpoint: [\[DELETE\] /v1/config/:kind/:name](/api-docs/config#delete-configuration)
+
 The `config delete` command deletes the configuration entry specified by the
 kind and name. See the [configuration entries docs](/docs/agent/config-entries)
 for more details about configuration entries.
 
+The table below shows this command's [required ACLs](/api#authentication). Configuration of
+[blocking queries](/api/features/blocking) and [agent caching](/api/features/caching)
+are not supported from commands, but may be from the corresponding HTTP endpoint.
+
+| ACL Required<sup>1</sup>                                      |
+| ------------------------------------------------------------- |
+| `service:write`<br />`operator:write`<br />`intentions:write` |
```

Comment:
Do you think we should have `or` between these 3 (possible) required permissions?

Reply:
I matched the style in the API docs.  I liked the `or` originally, but I think it makes sense like this given the details immediately below.  I don't want someone confusing `or` to globally mean one or the other.